Generator/Sources/NeedleFramework/Parsing/Pluginized/Processors/NonCoreComponentLinker.swift

```diff
@@ -39,13 +39,19 @@ class NonCoreComponentLinker: Processor {
         var nonCoreMap = [String: ASTComponent]()
         for nonCoreComponent in nonCoreComponents {
             nonCoreMap[nonCoreComponent.name] = nonCoreComponent
+            if nonCoreComponent.isLeaf {
+                nonCoreComponent.isLeaf = false
+            }
         }
 
         for pluginizedComponent in pluginizedComponents {
             guard let nonCoreComponent = nonCoreMap[pluginizedComponent.nonCoreComponentType] else {
                 throw GenericError.withMessage("Cannot find \(pluginizedComponent.data.name)'s non-core component with type name \(pluginizedComponent.nonCoreComponentType)")
             }
 
+            if pluginizedComponent.data.isLeaf {
+                pluginizedComponent.data.isLeaf = false
+            }
```

Comment:
Similar to above, I didn't understand this change.

Reply:
Added a comment